src/sentry/models/notificationsetting.py

```diff
@@ -104,6 +104,10 @@ def provider_str(self) -> str:
         ),
         null=False,
     )
+    is_migrated = models.BooleanField(
```

Comment:
@snigdhas if we put this in the code, Django is going to select for this column. Since this is a dangerous migration, it would break things unless Ops ran the migration BEFORE we deploy. why is it that you want to mark this as dangerous?

Reply:
fixed.